content/doc/pipeline/index.adoc

```diff
@@ -1,981 +1,437 @@
+== Why Pipeline?
 
-Jenkins Pipeline is a powerful, first-class feature for managing complex, multi-step pipelines. Jenkins Pipeline, a set of open source plugins and integrations, brings the power of Jenkins and the plugin ecosystem into a scriptable Domain Specific Language (DSL). Best of all, like Jenkins core, Pipeline is extensible by third-party developers, supporting custom extensions to the Pipeline DSL and various options for plugin integration.
+While standard Jenkins “freestyle” jobs support simple continuous integration by allowing you to define sequential tasks in an application lifecycle, they do not create a persistent record of execution, enable one script to address all the steps in a complex workflow, or confer the other advantages of pipelines.
 
-Jenkins Pipeline was known as Jenkins Workflow prior to version 1.13.
+In contrast to freestyle jobs, pipelines enable you to define the whole application lifecycle.  Pipeline functionality helps Jenkins to support continuous delivery (CD). The Pipeline plugin was built with community requirements for a flexible, extensible, and script-based CD workflow capability in mind. 
 
-![Pipeline Stage View UI](images/1-stage-view-ui-hr.png)
-Pipeline Stage View UI
+Accordingly, pipeline functionality is:
 
-This Refcard provides an overview and introduction to Jenkins Pipeline as well as a full syntax reference card. We also provide a real-world delivery pipeline example, building on the more basic Pipeline snippets from earlier examples.
+* Durable: Pipelines can survive both planned and unplanned restarts of your Jenkins master.
+* Pausable: Pipelines can optionally stop and wait for human input or approval before completing the jobs for which they were built.
+* Versatile: Pipelines support complex real-world CD requirements, including the ability to fork or join, loop, and work in parallel with each other.
+* Efficient: Pipelines can restart from any of several saved checkpoints.
+* Extensible: The Pipeline plugin supports custom extensions to its DSL (domain scripting language) and multiple options for integration with other plugins. 
 
-## Installing Jenkins Pipeline
-It is assumed you have already installed Jenkins – either via the CloudBees Jenkins Platform or jenkins-ci.org. Jenkins Version 1.609.1+ is required.
 
-* Open Jenkins in your web browser
-* Navigate to Manage Jenkins > Manage Plugins
-* Navigate to the Available tab, filter by Pipeline
-* Select the Pipeline plugin and install
-* Restart Jenkins
+The flowchart below is an example of one continuous delivery scenario enabled by the Pipeline plugin:
 
-This Refcard was written using Pipeline version 1.13. Installing the Pipeline plugin installs all necessary Pipeline dependencies and a new job type called Pipeline.
+image::/images/pipeline/jenkins-workflow.png[title="Jenkins Workflow", 800]
```

Comment:
The title on the image should be "Jenkins Pipeline", what's the going rate for saying "Workflow" when you mean "Pipeline" these days? $5? :smile_cat: